docs/analytics-ga4/ga4-ecommerce-tracker.md

```diff
@@ -0,0 +1,122 @@
+# Google Analytics ecommerce tracker (i.e. search)
+
+The ecommerce tracker script tracks search results in finder applications (e.g. search) and sends them to the `dataLayer`. There are 3 scenarios that result in a push to the `dataLayer`:
+
+- on page load
+- the application of filters and/or sorting
+- clicking a search result
+
+## Basic use
+
+In `analytics-ga4.js`:
+
+```JavaScript
+//= require ./analytics-ga4/ga4-ecommerce-tracker
+```
+
+In `live_search.js` in the `finder-frontend` repository (https://github.com/alphagov/finder-frontend/blob/main/app/assets/javascripts/live_search.js#L129):
+
+```JavaScript
+GOVUK.analyticsGa4.Ga4EcommerceTracker.init(isNewPage) // isNewPage is a boolean and is used to detect whether a new page has been loaded or not
+```
+
+Unlike the other GA4 tracking scripts in this repository, ecommerce tracking is not initialised by `init-ga4.js`. Instead, it is initialised by the JS that runs on finder pages (specifically, the `GOVUK.Ga4EcommerceTracking()` function). It is called:
```

Comment:
Is `GOVUK.Ga4EcommerceTracking()` correct?

Reply:
It isn't, it should say `LiveSearch.prototype.Ga4EcommerceTracking()` as you rightly pointed out